schema.sql

```diff
@@ -9,4 +9,26 @@ CREATE TABLE animal (
     weight_kg decimal
 );
 
-Alter TABLE animal ADD species char(20);
\ No newline at end of file
+Alter TABLE animal ADD species char(20);
+
+
+CREATE TABLE owners (
+    id int GENERATED BY DEFAULT AS IDENTITY,
+    full_name varchar(100),
+    age decimal,
+    PRIMARY KEY (id)
+);
+
+
+CREATE TABLE species (
+    species_id int GENERATED BY DEFAULT AS IDENTITY,
```

Comment:
**[OPTIONAL]**

- You could just leave the column as `id` so that you could reference it as `species_id` on another table so this won't get you confused when querying your tables.